3 Basic NLP.ipynb

```diff
@@ -53,7 +53,19 @@
    "cell_type": "code",
    "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
I think it is not a good idea to have an output which shows that cltk is not found in your Python environment!

Reply:
You're absolutely right! The output got messed up bc I didn't edit the nb in venv (I just changed the WordTokenizer parts)